lib/tasks/update_resource_quantity.rake

```diff
@@ -0,0 +1,14 @@
+namespace :data do
+  desc 'Update resources with nil quantity'
+  task update_resource_quantity: :environment do
+    Resource.all.each do |resource|
+      resource.used = 0 unless resource.used.present?
+      resource.quantity = resource.used unless resource.quantity.present?
+      if resource.save
+        puts 'Success for resource with id ' + resource.id.to_s + ' and name ' + resource.name
+      else
+        puts 'Failed for resource with id ' + resource.id.to_s + ' and name ' + resource.name
```

Comment:
There is still no message at the end indicating that the task has fully run, how can the user running the task know that it has finished?